app/views/RoomsListView/index.tsx

```diff
@@ -899,7 +900,7 @@ class RoomsListView extends React.Component<IRoomsListViewProps, IRoomsListViewS
 				ref={this.getScrollRef}
 				data={searching ? search : chats}
 				extraData={searching ? search : chats}
-				keyExtractor={keyExtractor}
+				keyExtractor={item => keyExtractor(item, !!searching)}
```

Comment:
If you fix `searching` type, you can avoid `!!`.
I don't think `searching` is ever `undefined`.